pkg/synopsysctl/cmd_create.go

```diff
@@ -317,15 +317,15 @@ var createBlackDuckCmd = &cobra.Command{
 			if err != nil {
 				return err
 			}
-			defaultPvcComponentsList, err := app.Blackduck().GetComponents(&blackduckv1.Blackduck{ObjectMeta: metav1.ObjectMeta{Name: blackDuckName, Namespace: blackDuckNamespace}, Spec: blackDuckSpec}, blackduckapp.PVCResources)
+			defaultPvcComponentsList, err := app.Blackduck().GetComponents(&blackduckv1.Blackduck{Spec: blackDuckSpec}, blackduckapp.PVCResources)
 			if err != nil {
 				return err
 			}
 			defaultPvcList := []blackduckv1.PVC{}
 			for _, defaultPvcComponent := range defaultPvcComponentsList.PersistentVolumeClaims {
 				defaultPvcComponentResourceQuantitySize := defaultPvcComponent.Spec.Resources.Requests[v1.ResourceStorage]
 				pvc := blackduckv1.PVC{
-					Name: defaultPvcComponent.Name,
+					Name: defaultPvcComponent.Name[1:],
```

Comment:
Please add a unit test to avoid regression 😃

Reply:
I'm going to add the Unit Tests in a different PR